_posts/en/2017-03-13-performance-optimizations-1.md

```diff
@@ -0,0 +1,77 @@
+With that in mind, the block validation mechanism was re-engineered in order to be able to allocate script checks to parallel threads so that their verification can happen even before the client is done fetching all of the UTXOs from the block. To achieve this, script check actions are stored in a queue after transaction are processed and are handled separately from other input validation jobs. 
+
+As a consequence, synchronization to the tip of the chain happens much faster by making more efficient use of the peer’s resources. During testing of the implementation, developers noted 35% to 100% speed-up when benchmarking against previous versions of the software.
+
+##Headers first synchronization##
+Release: Bitcoin Core 0.10
+
+Striving to further improve initial block download time, the Core project introduced in late 2014 an important re-architecture of the mechanism used by nodes to synchronize with the “best chain”. 
+
+Initially, the process of bootstrapping a new Bitcoin client would involve a user fetching block data from a single peer with the consequence that any interruption or decrease in connection quality would significantly stall the process. With an ever-increasing blockchain size, this would result in sometimes massive waiting time for the synchronization to complete, with a large percentage of users reporting up to multi-day periods depending on their hardware.
+
+Headers first synchronization severely mitigates this issue using a new method that involves nodes first downloading and validating block headers from a single peer and then fetching block data in parallel from a multitude of others. 
+
+Complaints about initial block download time have been prevalent since the early days of Bitcoin. With headers first synchronization, the software took a major step forward in terms of usability for new users. Rather than wasting many hours on unreliable synchronization, nodes could now leverage their entire network of peers and cut down the bootstrapping time significantly. With the use of smarter algorithms, another asymptotic improvement was made to this crucial aspect of Bitcoin’s long term sustainability. 
+
+##Block file pruning##
+Release: Bitcoin Core 0.11
+
+Block file pruning was first described by Satoshi Nakamoto in his white paper as a potential solution to disk space scarcity. Seven years later, with the blockchain reaching hundreds of gigabytes, the ability to delete raw blocks of old data after validation came as a major boon to users with limited resources.
```

Comment:
Added content according to your suggestions and adjusted the paragraph in question. Let me know if that works. Thanks for reviewing!